### PR TITLE
Update CLI Docs

### PR DESCRIPTION
**Pull request by sambukowski**
## Description
Update the cli tutorial sections for the go cli.
Bring those same sections up to date for use with Dusk-5

### CODE REVIEW

File: docs/developer/tutorials/install-the-cli.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Install the Astria CLI
+
```
Comment (quasystaty1) [inline docs/developer/tutorials/install-the-cli.md:2]:
```suggestion

The astria-go cli is a tool designed to make local rollup development as simple and dependency free as possible. It provides functionality to easily run the Astria stack and interact with the Sequencer.
```

File: docs/developer/tutorials/install-the-cli.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+cd astria-cli-go
+just build
+
+# run the cli in the repo directory
+just run "version"
+
+# OR you can move the binary to a location in your PATH if you'd like
+mv ./bin/astria-go /usr/local/bin/
+astria-go version
+```
```
Comment (quasystaty1) [inline docs/developer/tutorials/install-the-cli.md:27]: Maybe adding available commands table here?

**Review by quasystaty1 — APPROVED** [review]
Section on common issues is great addition!
LGTM

**Comment by jbowen93** [comment]
We should update the installing go-ethereum component to use github.com/astriaorg/astria-geth instead of github.com/astriaorg/go-ethereum

**Comment by jbowen93** [comment]
for local rollup and local sequencer instructions there is no step to specify a custom config for geth
```
 "config": {
        "chainId": <6 digit number>,
				...
        "astriaRollupName": "<rollup name>"
    },
 ```
 this means that when we get to the test transactions phase the user doesn't know the chain-id of their geth node

**Comment by sambukowski** [comment]
> We should update the installing go-ethereum component to use github.com/astriaorg/astria-geth instead of github.com/astriaorg/go-ethereum

It is update to this for all the go-cli stuff, but not for the rest of the existing docs.

**Comment by sambukowski** [comment]
> for local rollup and local sequencer instructions there is no step to specify a custom config for geth

If they are just running things locally and pulling the repos for local everything, they don't need to update these. The default chain id and rollup name when pulling geth and the test transactions are the same.

I'll add that section though, to avoid confusion.

**Review by quasystaty1 — APPROVED** [review]

**Review by jbowen93 — APPROVED** [review]

**Pull request merged**